ArcGISRuntimeSDKQt_CppSamples/Maps/OpenMapUrl/README.md

```diff
@@ -20,12 +20,14 @@ A web map can be selected from the drop-down list. On selection the web map disp
 
 ## Relevant API
 
-* Map
+* ArcGISMap
```

Comment:
`ArcGISMap` is the name of the class in a different API (Java, I believe). Our equivalent is `Map`.
```suggestion
* Map
```